src/app/file-upload-modal/file-upload-modal.component.html

```diff
@@ -0,0 +1,62 @@
+<header>
+  <h3>{{ modalData.title || "Select file(s)" }}</h3>
+</header>
+<main class="close-warning" *ngIf="this.closeDialog">
+  <h4>Are you sure you wish to close?</h4>
+  <p>No files will be uploaded and changes to files will not be saved.</p>
+  <div class="close-buttons">
+    <button class="btn btn-secondary" (click)="handleCancelClose()">Cancel</button>
```

Comment:
Is it super clear to the end user, what option they should pick... Close / cancel.  just saying sometimes i get confused with these types of messages...

Reply:
I had thought about that yeah. Although I'm not a UI designer. LUP users should be used to secondary and primary button colouring for "stop" and "go" respectively. I tried to use that same colour pattern here to help.